### PR TITLE
Gough 5-6

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: history-of-the-quakers-v3/original/25-book-6-chapter-5.adoc [hunk 1]
```diff
@@ -212,9 +215,6 @@ to impose upon them, and tempt them to evil, are not principally accountable,
 for the crimes they commit, and their consequences,
 is a query worthy of their most serious consideration.
 {footnote-paragraph-split}
-From Maija:
-Here`'s the footnote within a footnote
-{footnote-paragraph-split}
 *At the treaty at Carlisle,
```
Comment (maijapapaya) [inline history-of-the-quakers-v3/original/25-book-6-chapter-5.adoc:218]: Here's a footnote within a footnote that I marked off with an asterisk.

File: history-of-the-quakers-v3/original/26-book-6-chapter-6.adoc [hunk 2]
```diff
@@ -306,6 +310,9 @@ than to be over curious in questions of words, ministering to strife and content
 reminding them of the ancient and constant principle and experience of friends,
 that the dispensation of the gospel committed to them, was a
 
+[.embedded-content-document.epistle]
```
Comment (maijapapaya) [inline history-of-the-quakers-v3/original/26-book-6-chapter-6.adoc:313]: So this begins quoting the epistle midway through and it looks a little funny when it's embedded so idk if there's anything we can do to make it look better...ellipses maybe?? Idk.

File: history-of-the-quakers-v3/original/25-book-6-chapter-5.adoc (hunk 1 above)
Comment (jaredh159) [reply history-of-the-quakers-v3/original/25-book-6-chapter-5.adoc:218]: Ok, this actually turned out to look fairly decent in the PDF, so I think we can leave it.  However, lets escape it, so it's `+++*+++At the treaty at Carlisle,` instead 👍

File: history-of-the-quakers-v3/original/25-book-6-chapter-5.adoc [hunk 3]
```diff
@@ -413,15 +417,25 @@ as you are not and cannot be forgotten in my addresses and approaches to him,
...
 
-2nd 1st mo 1689
-
+[.signed-section-signature]
 William Penn
 
+[.signed-section-context-close]
+2nd 1st mo 1689
+
+[.signed-section-context-close]
```
Comment (jaredh159) [inline history-of-the-quakers-v3/original/25-book-6-chapter-5.adoc:426]: I sort of lean towards making this a postscript, although it doesn't fit perfectly in either category.

File: history-of-the-quakers-v3/original/26-book-6-chapter-6.adoc (hunk 2 above)
Comment (jaredh159) [reply history-of-the-quakers-v3/original/26-book-6-chapter-6.adoc:313]: Yeah, lets add an ellipses, like this (note the lack of space after it):

```ASCIIDOC
&hellip;spiritual dispensation; in nowise to oppose,
```

**Comment by jaredh159** [comment]
thanks Maija, merged. 👍

**Pull request merged**